sonic-net/sonic-buildimage
# [FS][arm64] support new boards s5800-48t4s and s5800-48t8s-mars8p

#### Why I did it
   Adding platform support for FS s5800-48t4s and s5800-48t8s-mars8p.
   Both s5800-48t4s and s5800-48t8s-mars8p have 48 * 10/100/1000 Base-T ports, 4 * 10GE SFP+ Ports on Centec TsingMa.
   s5800-48t4s is different from s5800-48t8s-mars8p in that:
   The phy chip used by s5800-48t4s is Marvell 88e1680;
   The phy chip used by s5800-48t4s-mars8p is Centec ctc21108;


#### How I did it
Modify related code in platform and device directory.
#### How to verify it
Build amd64 sonic image, verify platform functions (port, sfp, led etc) on FS  s5800-48t4s and s5800-48t8s-mars8p board.
#### Which release branch to backport (provide reason below if selected)




#### Description for the changelog


#### Ensure to add label/tag for the feature raised. example - PR#2174 under sonic-utilities repo. where, Generic Config and Update feature has been labelled as GCU.

#### Link to config_db schema for YANG module changes


#### A picture of a cute animal (not mandatory but encouraged)
